src/test/java/com/github/mauricioaniche/ck/MethodInvocationsTest.java

```diff
@@ -0,0 +1,208 @@
+package com.github.mauricioaniche.ck;
+
+import com.google.common.collect.Sets;
+import junit.framework.Assert;
+import org.apache.commons.lang.ArrayUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class MethodInvocationsTest extends BaseTest {
```

Comment:
Some extra test cases I always think about:
- method inside if blocks (to make sure the implementation handles nested blocks well)
- lambda expressions
- anonymous classes within a method (does it count correctly? or at least, do you understand what's gonna happen if there's an anonymous class)
- method call in the `return` expression (e.g., `return a();`)
- static call invocation

Reply:
Thanks for the advice, the additional tests discovered a new bug.